core/src/epicli/cli/helpers/doc_list_helpers.py

```diff
@@ -22,5 +28,5 @@ def select_single(documents, query):
         elements_count = len(results)
         if elements_count == 1:
             return results[0]
-        raise Exception("Expected one element but received: " + str(elements_count))
+        raise ExpectedSingleResultException("Expected one element but received: " + str(elements_count))
```

Comment:
shouldn't that be tested as well?

Reply:
Working on it! :+1:

Reply:
Missing tests for the `select_single` helper added in this commit https://github.com/epiphany-platform/epiphany/pull/1359/commits/bdccd06ec4c99f37acbd5e790e0207375338deae :+1: